File: Assets/Smash Way/Scripts JS/GameOther/RecordUse.js
#pragma strict

var FinishObj:GameObject;
private var Distance:int;
var Sprite:GameObject;

function Awake () {
Distance=PlayerPrefs.GetInt(FinishObj.GetComponent.<Finish>().AssembleName.ToString());
gameObject.transform.position.z=Distance;
if(FinishObj.GetComponent.<Finish>().NewAssembleL<=PlayerPrefs.GetInt("AssembleL")){
gameObject.SetActive(false);
}
}

function Start(){
yield WaitForSeconds(7);
Sprite.SetActive(true);
}

File: Assets/Smash Way/Scripts CS/GameOther/RecordUse.cs
using UnityEngine;
using System.Collections;

public class RecordUse : MonoBehaviour {



	public GameObject FinishObj;
	private int Distance;
	public GameObject Sprite;

	void Awake () {
		Distance=PlayerPrefs.GetInt(FinishObj.GetComponent<Finish>().AssembleName.ToString());
		gameObject.transform.position = new Vector3 (gameObject.transform.position.x,gameObject.transform.position.y,Distance);
		if(FinishObj.GetComponent<Finish>().NewAssembleL<=PlayerPrefs.GetInt("AssembleL")){
			gameObject.SetActive(false);
		}
	}

	void Start(){
		StartCoroutine ("iStart");
	}

	IEnumerator iStart(){
		yield return new WaitForSeconds(7);
		Sprite.SetActive(true);
	}
}
